config/environments/deployed_development.rb

```diff
@@ -89,10 +89,27 @@
   # require 'syslog/logger'
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger           = ActiveSupport::Logger.new(STDOUT)
-    logger.formatter = config.log_formatter
-    config.logger    = ActiveSupport::TaggedLogging.new(logger)
+  # Logging
+  config.log_level = :debug # Debug logging in dev
+  config.log_tags = [:request_id] # Prepend all log lines with the following tags.
+  logger = ActiveSupport::Logger.new(STDOUT)
+  logger.formatter = config.log_formatter
+  config.logger = ActiveSupport::TaggedLogging.new(logger)
+
+  # Use Lograge for cleaner logging
+  config.lograge.enabled = true
+  config.lograge.base_controller_class = ["ActionController::API", "ActionController::Base"]
+  config.lograge.formatter = Lograge::Formatters::Logstash.new
+  config.lograge.ignore_actions = ["ApplicationController#check"]
+  config.lograge.logger = ActiveSupport::Logger.new($stdout)
+
+  # Include params in logs: https://github.com/roidrage/lograge#what-it-doesnt-do
+  config.lograge.custom_options = lambda do |event|
+    exceptions = %w[controller action format id]
```

Comment:
Are there any secure tokens or passwords that we might want to exclude here?

Reply:
Hmm, all I can think of is `login_token`, which I guess we should hide - it isn't very long lived, but if someone failed to log in with a valid token due to some exception, and someone had the logs, and they tried to log in with the same token, they might be able to get in.

Other than that, I don't think we have anything relating to security in params.

Reply:
Actually, we add 
```
Rails.application.config.filter_parameters += %i[
  passw secret token _key crypt salt certificate otp ssn
]
```
in one of our initialisers, and https://guides.rubyonrails.org/configuring.html tells me it works "by partial matching regular expression", so anything with a `token` should be hidden already.